code/game/objects/items/stacks/rods.dm

```diff
@@ -9,7 +9,7 @@
 	throwforce = 15.0
 	throw_speed = 5
 	throw_range = 20
-	matter = list(DEFAULT_WALL_MATERIAL = 1875)
+	matter = list(DEFAULT_WALL_MATERIAL = SHEET_MATERIAL_AMOUNT / 2)

```

Comment:
It really doesn't matter. This macro is 2000. This code doesn't cause any issues currently. If someone is going to change the macro, it will be their responsibility to make sure it doesn't cause any issues.